src/main/java/Kratos.java

```diff
@@ -0,0 +1,187 @@
+    static Task[] tasksList = new Task[MAX_TASKS];
+
+    public static int count = 0;
+
+    // Method to greet the user
+    public static void greet() {
+        String logo = "──────────────────────────────\n" +
+                "───────────────────────▓▓▓▓───\n" +
+                "──────────────────────▓▓▓▓▓▓──\n" +
+                "─────────────────────▓▓▓▓▓▓▓▓─\n" +
+                "────────────────────▓▓▓▓▓▓▓▓──\n" +
+                "───────────────────▓▓▓▓▓▓▓▓───\n" +
+                "──────────────────▓▓▓▓▓▓▓▓────\n" +
+                "─────────────────▓▓▓▓▓▓▓▓─────\n" +
+                "─────────█──────▓▓▓▓▓▓▓▓──────\n" +
+                "────────██─────▓▓▓▓▓▓▓▓───────\n" +
+                "───────██──────▓▓▓▓▓▓▓▓▓──────\n" +
+                "─▀██▄─██───────▓▓▓▓▓▓▓▓██──▄█─\n" +
+                "─█████████▄─▄───▓▄▓▄████████──\n" +
+                "──█▀▀▀███████────███████▀▀▀██─\n" +
+                "─███▄▄██▄███▀─────███▄██▄▄███─\n" +
+                "─█─██████▀█────────▓████████▀─\n" +
+                "────██▀───▀─█────█──▓▓▓▓▓▓▓▓──\n" +
+                "────▄█▀──────█──█────▓▓▓▓▓▓▓▓─\n" +
+                "───▀█────────█──█─────▓▓▓▓▓▓▓─\n" +
+                "───▄█▀────▄▄█▀──▀█▄▄───▓▓▓▓▓▓─\n" +
+                "───█▄────█──█────█──█──▓▓▓▓▓▓─\n" +
+                "───█─────▀──────────▀──▓▓▓▓▓▓─\n" +
+                "────▀─────▀█──────█▀────▓▓▓───\n" +
+                "────────────▀▄▄▄▄▀──────▓▓────\n" +
+                "────────────────────────▓─────\n" +
+                "────────▄█████████████▄───────\n" +
+                "───────██▀▀▀▀▀▀▀▀▀▀▀▀▀██──────\n" +
+                "───────▀───────────────▀──────\n" +
+                "───────────█████████──────────\n" +
+                "────────────███████───────────\n" +
+                "──────────────███─────────────\n" +
+                "──────██▄█─▄─█████─▄─█▄██─────\n" +
+                "───────█████████████████──────\n" +
+                "────────███████████████───────\n" +
+                "─────────█████████████────────\n" +
+                "──────────███████████─────────\n" +
+                "──────────▀─███████─▀─────────\n" +
+                "────────────▀─███─▀───────────\n" +
+                "───────────────▀──────────────\n";
+        System.out.println(logo);
+        System.out.println(LINE);
+        System.out.println("Kratos commends you for your presence. Prepare for battle.\n" +
+                        "Enter your commands with purpose.");
+        System.out.println(LINE);
+    }
+
+    // Method to say goodbye
+    public static void end() {
+        System.out.println(LINE);
+        System.out.println("            Until the next battle, mortal.\n" +
+                "May your tasks be conquered with the ferocity of a god.");
+        System.out.println(LINE);
+    }
+
+    // Method to display marking of tasks
+    public static void displayMarking(int taskNumber, String mark) {
```

Comment:
method name could be a bit misleading! perhaps could change this for better readability (e.g. because i think that this method sort of identifies whether the task should be marked or not and then mark/unmark the task accordingly, while also displaying the message outputted from each operations!). a suggestion could be performMarkOperations() or anything else appropriate :]?